app/src/main/java/com/gotenna/app/home/HomeViewModel.kt

```diff
@@ -571,73 +570,15 @@ class HomeViewModel : ViewModel() {
     }
 
     fun scanChannels() {
-        uiScope.launch {
```

Comment:
Not great, but we will deal with this later as part of the flavors of the sample app.

Reply:
I mean, since this is just a copy of what we have internally we will need to manually commit to here. As this is no longer in the external library.